generator/src/main/java/com/faforever/neroxis/generator/style/StyleGenerator.java

```diff
@@ -56,7 +56,7 @@ public abstract class StyleGenerator implements HasParameterConstraints {
     private PropGenerator propGenerator;
     private DecalGenerator decalGenerator;
     private SCMap map;
-    private Random random;
+    protected Random random;
```

Comment:
Why was the random made protected? This is a bit risky and should be avoided due to determnism